.snyk

```diff
@@ -0,0 +1,8 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.22.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-ORGJETBRAINSKOTLIN-2393744:
+    - '*':
+        reason: no available replacement
+        expires: 2022-08-31T00:00:00.000Z
```

Comment:
I think you need to extend this a bit more. Do 3 months from now.

Reply:
done